### PR TITLE
New Package: Steam Locomotive (sl)

### PR DESCRIPTION
**Pull request by scaryrawr**
Checklist for Pull Requests
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

Adds an installer for [SL](https://github.com/scaryrawr/sl). Even Windows needs command line trains.
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/193760)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 899.)

File: manifests/s/ScaryRawr/SL/5.06.0/ScaryRawr.SL.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+UpgradeBehavior: uninstallPrevious
+Commands:
+- sl
+Installers:
```
Comment (stephengillie) [inline manifests/s/ScaryRawr/SL/5.06.0/ScaryRawr.SL.installer.yaml:17]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Manual Validation ended with:
image

File: manifests/s/ScaryRawr/SL/5.06.0/ScaryRawr.SL.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+PackageIdentifier: ScaryRawr.SL
+PackageVersion: 5.06.0
+PackageLocale: en-US
+Publisher: ScaryRawr
+PublisherUrl: https://github.com/scaryrawr/sl
+PublisherSupportUrl: https://github.com/scaryrawr/sl/issues
+# PrivacyUrl:
+# Author:
+PackageName: Steam Locamotive (SL)
```
Comment (scaryrawr) [inline manifests/s/ScaryRawr/SL/5.06.0/ScaryRawr.SL.locale.en-US.yaml:12]:
```suggestion
PackageName: Steam Locomotive (SL)
```

File: manifests/s/ScaryRawr/SL/5.06.0/ScaryRawr.SL.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+License: MIT
+LicenseUrl: https://github.com/scaryrawr/sl/blob/main/LICENSE
+# Copyright:
+# CopyrightUrl:
+ShortDescription: Prints a steam locomotive if you type sl instead of ls
+# Description:
+Moniker: sl
+Tags:
+- sl
+- steam-locamotive
```
Comment (scaryrawr) [inline manifests/s/ScaryRawr/SL/5.06.0/ScaryRawr.SL.locale.en-US.yaml:23]:
```suggestion
- steam-locomotive
```

File: manifests/s/ScaryRawr/SL/5.06.0/ScaryRawr.SL.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS0.CRLF.7-5-0-rc-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: ScaryRawr.SL
+PackageVersion: 5.06.0
+PackageLocale: en-US
+Publisher: ScaryRawr
```
Comment (Trenly) [inline manifests/s/ScaryRawr/SL/5.06.0/ScaryRawr.SL.locale.en-US.yaml:7]:
```suggestion
Publisher: sl
```
This needs to match Control Panel, or `AppsAndFeaturesEntries` need to be specified in order for WinGet to properly detect the package

File: manifests/s/ScaryRawr/SL/5.06.0/ScaryRawr.SL.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+PublisherUrl: https://github.com/scaryrawr/sl
+PublisherSupportUrl: https://github.com/scaryrawr/sl/issues
+# PrivacyUrl:
+# Author:
+PackageName: Steam Locomotive (SL)
```
Comment (Trenly) [inline manifests/s/ScaryRawr/SL/5.06.0/ScaryRawr.SL.locale.en-US.yaml:12]:
```suggestion
PackageName: sl
```
This needs to match Control Panel, or AppsAndFeaturesEntries need to be specified in order for WinGet to properly detect the package

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**